src/main/java/io/anserini/analysis/vectors/NearestVector.java

```diff
@@ -0,0 +1,147 @@
+package io.anserini.analysis.vectors;
```

Comment:
How about rename the class to something that aligns closer to the demo paper, like `ApproximateNearestNeighborSearch`?

Reply:
+1